conda/cli/main_clean.py

```diff
@@ -1,41 +1,43 @@
 # Copyright (C) 2012 Anaconda, Inc
 # SPDX-License-Identifier: BSD-3-Clause
+from __future__ import annotations
 
 from logging import getLogger
 from os import lstat, walk
 from os.path import isdir, join
-from typing import Any, Dict, Iterable, List, Tuple
+from typing import Any, Iterable
 import sys
 
 from ..base.constants import CONDA_PACKAGE_EXTENSIONS, CONDA_TEMP_EXTENSIONS, CONDA_LOGS_DIR
 from ..base.context import context
 
 log = getLogger(__name__)
-_EXTS = (*CONDA_PACKAGE_EXTENSIONS, *(f"{e}.part" for e in CONDA_PACKAGE_EXTENSIONS))
+_EXTENSIONS = (*CONDA_PACKAGE_EXTENSIONS, *(f"{ext}.part" for ext in CONDA_PACKAGE_EXTENSIONS))
```

Comment:
Do we use `.part` while downloading packages? It might be a good idea but I don't recall the code that does so.

Reply:
not sure